demo/index.html

```diff
@@ -73,14 +73,9 @@
         --paper-icon-button-ink-color: var(--paper-indigo-500);
       }
 
-      paper-icon-button.huge::shadow #icon {
-        width: 100px;
-        height: 100px;
-      }
-
-      paper-icon-button.huge #icon {
-        width: 100px;
-        height: 100px;
+      paper-icon-button.huge {
+        --iron-icon-height: 100px;
+        --iron-icon-width: 100px;
```

Comment:
Does this actually work? I thought that this had to be assigned in a `:host` or `:root`..